js/index.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+import Gladiator from './gladiator.js';
+import Arena from './arena.js';
+
+
+
+
+
+
+// //part 1 Gladiator
+// const max = new Gladiator("Maximus", "Trident");
+// console.log(max.name); // "Maximus"
+// console.log(max.weapon); // "Trident"
+
+// // part 2 #1
+// const colosseum = new Arena("Colosseum");
+// console.log(colosseum.name); // => Colosseum
+
+// // part 2 #2
+// const megalopolis = new Arena("megalopolis");
+// console.log(megalopolis.name.charAt(0).toUpperCase() + megalopolis.name.slice(1)); // => Megalopolis
```

Comment:
put this line in the gladiator class file and you will never have to do this every time you have a new gladiator

```
this.name.charAt(0).toUpperCase() + megalopolis.name.slice(1)
```